DxDispatch/src/dxdispatch/PixCaptureHelper.cpp

```diff
@@ -14,10 +14,14 @@ static std::wstring Utf8ToWideString(std::string_view utf8String)
     {
         return std::wstring();
     }
+    else if (uint64_t(utf8String.size()) > std::numeric_limits<int32_t>::max())
```

Comment:
nit: don't mix c-style cast with c++-style cast.

Reply:
At least it's constructor-style C++ casting and not C-style casting ;b `(uint64_t)(utf8String.size())`.

Reply:
My eyes totally missed that. :)